modules/apps/site-initializer/site-initializer-cms/src/main/resources/site-initializer/layouts/03_assets/01_all/page-definition.json

```diff
@@ -0,0 +1,28 @@
+{
+	"pageElement": {
+		"id": "cf15ab1c-5f86-60e6-aa6e-5c1e44a5c2a9",
+		"pageElements": [
+			{
+				"definition": {
+					"fragment": {
+						"key": "com.liferay.site.cms.site.initializer.internal.fragment.renderer.AllSectionFragmentRenderer"
+					},
+					"fragmentConfig": {
+					},
+					"fragmentFields": [
+					],
+					"indexed": true
+				},
+				"id": "2435c940-ab4e-4bfe-d04b-af7744ec304f",
+				"type": "Fragment"
+			}
+		],
+		"type": "Root"
+	},
+	"settings": {
+		"masterPage": {
+			"key": "cms-master"
+		}
+	},
+	"version": 1.1
```

Comment:
Why are we starting the new page-definition on version 1.1?

Reply:
To generate these files, I am using the export feature in the portal, as @jkappler showed me. I do not know why, but it starts the version at 1.1

Reply:
After speaking with Jürgen regarding this, version 1.1 is required at the moment, otherwise we use legacy code which does not work properly.